packages/paypal-commerce-integration/src/paypal-commerce-alternative-methods/create-paypal-commerce-alternative-methods-payment-strategy.ts

```diff
@@ -14,7 +14,9 @@ const createPayPalCommerceAlternativeMethodsPaymentStrategy: PaymentStrategyFact
     new PayPalCommerceAlternativeMethodsPaymentStrategy(
         paymentIntegrationService,
         createPayPalCommerceIntegrationService(paymentIntegrationService),
-        new LoadingIndicator({ styles: { backgroundColor: 'black' } }),
+        new LoadingIndicator({
+            containerStyles: { 'background-color': 'rgba(0, 0, 0, 0.4)', 'z-index': '1000' },
```

Comment:
`{ 'background-color': 'rgba(0, 0, 0, 0.4)', 'z-index': '1000' }` appears three times in the PR.
Could we set it as a constant and put it in a separate file, making it easier to manage the look?

Reply:
Updated